### DISCUSSION TITLE
Hovering over the top menu scrolls the left menu in editor when expecting to scroll through the timeline

### DISCUSSION THREAD

**Discussion by Okorin** [ppy/osu#30140] [Q&A] [answered]
When in Edit mode and hovering over the timeline, scrolling will move it back and forth. 
When you move your cursor in the top bar with the logo and the two menu bars (Song setup / file / etc. & timing etc on the right), it will scroll the left part of the menu instead. 

this is the part of the menu that moves
![this part](https://puu.sh/KgeFA/fe5ab4970d.png)

this makes scrolling through the editor require more precision than necessary IMO

**Comment by SupDos** [comment]
This entire section (in red) scrolls through the timeline

![image](https://github.com/user-attachments/assets/1710beb3-c134-40bb-b855-35f2f992cb1e)

Is more precision than this really needed...?

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/30141
